workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -142,19 +142,19 @@ protected virtual LocatorConfig GetLocatorConfig(string workerType)
                 catch (ConnectionFailedException e)
                 {
                     logger.HandleLog(LogType.Error,
-                        new LogEvent($"Failed attempt to create worker")
+                        new LogEvent($"Failed to create worker")
```

Comment:
Still feels like "an attempt" failed. 

Spinoffs below:

Should that become a warning instead of an error?

Should it print out how many attempts are left / which attempt it was?

If there are 3 failed attempts the user will see:

- "{error} Failed to create worker"
- "{error} Failed to create worker"
- "{error} Failed to create worker"
- "{exception} Tried to connect 3 times - giving up"

Doesn't look too bad actually, but still something to consider.

Reply:
The main reason I had attempt in here before is because the Task<Worker> hasn't failed. It just hasn't finished yet. The latter message that said it failed to create a worker meant that the potential worker really was dead and should be removed. I agree that a warning might be better here, which would itself make it clear that it might still succeed.